packages/commonwealth/client/scripts/views/components/Mava/Mava.scss

```diff
@@ -3,7 +3,7 @@
 #mava {
   .loadButton {
     @media only screen and (max-width: $breakpoint-small-max-px) {
-      bottom: 75px !important;
+      display: none !important;
```

Comment:
it should not be visible anymore?

Reply:
no... we're removing it per @dillchen and I have approval to get it nuked in this PR.  Was slated for the next PR but it conflicts with the sticky editor now.

Reply:
so let's remove it completely with script etc as now it looks like you just hiding it